cron/cron.ts

```diff
@@ -21,18 +21,18 @@ export async function monthly() {
   await Model.PrayTime.deleteMany()
 
   for (let i = 0; i < regions.length; i++) {
-    const pdf = await axios.get(process.env.TIME_API + regionIds[i] + '/' + currentMonth, {
+    const pdf = await axios.get(String(process.env.TIME_API) + regionIds[i] + '/' + currentMonth, {
       responseType: 'arraybuffer',
     })
-    const pdfData = await pdfParser(pdf.data, { normalizeWhitespace: true })
```

Comment:
why u deleted {normalizeWhiteSpace: true}

Reply:
there's not an option named normalizeWhiteSpace